addons/packages/load-balancer-and-ingress-service/1.4.3/README.md

```diff
@@ -6,9 +6,220 @@ This package provides NSX Advanced Load Balancer.
 
 * load-balancer-and-ingress-service
 
+## Supported Providers
+
+The following tables shows the providers this package can work with. Other cloud provider support will be added  
+in the future.
+
+| AWS  |  Azure  | vSphere  | Docker |
```

Comment:
Actually AKO and AVI are supported in public cloud, we should correct this, it is supported.